public/app/plugins/panel/heatmap-new/fields.ts

```diff
@@ -106,7 +107,12 @@ const getSparseHeatmapData = (
     };
   }
 
+  // y axis tick label display
+  updateFieldDisplay(frame.fields[1], options.yAxis, theme);
+
+  // cell value display
   const disp = frame.fields[3].display ?? getValueFormat('short');
+
```

Comment:
maybe we should also do:

  const disp = updateFieldDisplay(frame.fields[3].display, options.cellValues, theme);